components/navbar.tsx

```diff
@@ -33,13 +45,23 @@ export default async function Navbar() {
         >
           Home
         </Link>
-        <a
-          href="#userTestimonials"
-          className="text-navbarSecondary hover:text-primary font-semibold transition md:block hidden dark:text-white"
-          style={{ scrollBehavior: "smooth" }}
-        >
-          Testimonial
-        </a>
```

Comment:
One line fix: revert this component and update the `href` to `href="/#userTestimonials"`. Also, change the `<a>` tag to use `<Link>` from `next/link`.

Reply:
Implement this @Shahbaz898414

Reply:
ok